tests/unit/script/size_of.rs

```diff
@@ -30,13 +30,13 @@ macro_rules! sizeof_checker (
 
 // Update the sizes here
 sizeof_checker!(size_event_target, EventTarget, 40);
-sizeof_checker!(size_node, Node, 152);
-sizeof_checker!(size_element, Element, 328);
-sizeof_checker!(size_htmlelement, HTMLElement, 344);
-sizeof_checker!(size_div, HTMLDivElement, 344);
-sizeof_checker!(size_span, HTMLSpanElement, 344);
-sizeof_checker!(size_text, Text, 184);
-sizeof_checker!(size_characterdata, CharacterData, 184);
+sizeof_checker!(size_node, Node, 184);
+sizeof_checker!(size_element, Element, 360);
+sizeof_checker!(size_htmlelement, HTMLElement, 376);
+sizeof_checker!(size_div, HTMLDivElement, 376);
+sizeof_checker!(size_span, HTMLSpanElement, 376);
+sizeof_checker!(size_text, Text, 216);
+sizeof_checker!(size_characterdata, CharacterData, 216);
```

Comment:
According to TravisCI, these numbers still aren't right. Maybe someone else updated the types in a different commit:
```
	thread 'size_of::size_div' panicked at 'Your changes have decreased the stack size of commonly used DOM struct HTMLDivElement from 376 to 368. Good work! Please update the size in tests/unit/script/size_of.rs.', /home/travis/build/servo/servo/tests/unit/script/size_of.rs:36
	thread 'size_of::size_element' panicked at 'Your changes have decreased the stack size of commonly used DOM struct Element from 360 to 352. Good work! Please update the size in tests/unit/script/size_of.rs.', /home/travis/build/servo/servo/tests/unit/script/size_of.rs:34
	thread 'size_of::size_htmlelement' panicked at 'Your changes have decreased the stack size of commonly used DOM struct HTMLElement from 376 to 368. Good work! Please update the size in tests/unit/script/size_of.rs.', /home/travis/build/servo/servo/tests/unit/script/size_of.rs:35
thread 'size_of::size_span' panicked at 'Your changes have decreased the stack size of commonly used DOM struct HTMLSpanElement from 376 to 368. Good work! Please update the size in tests/unit/script/size_of.rs.', /home/travis/build/servo/servo/tests/unit/script/size_of.rs:37
```